src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < 6; i++) {
+            figures[i] = supplier.getRandomFigure();
```

Comment:
for the last 3 you should generate default figure

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/d9516e88-a457-4fff-b8a9-90f95c995825)